core/src/main/java/org/eclipse/hono/config/AbstractConfig.java

```diff
@@ -41,25 +41,23 @@
     /**
      *  A logger to be shared with subclasses.
      */
-    protected final Logger LOG = LoggerFactory.getLogger(getClass());
+    public static final Logger LOG = LoggerFactory.getLogger(AbstractConfig.class);
```

Comment:
Can you explain the motivation here? With the old definition, there is the added info in the log from which actual (sub)class the log statements are coming.

Reply:
That was left from the PoC phase as there were some issues with native image. Now everything works fine, so I reverted it back.